### ISSUE TITLE
Does not work with dns.adguard.com as secure DNS provider in Chrome.

### ISSUE THREAD

**Issue by deNoor** [AdguardTeam/AdGuardVPNExtension#27]
**Steps to reproduce**
1. Install this extension v1.0.33 for Chrome.
2. Go to chrome Settings - DNS - enable Secure DNS with Custom provider as https://dns.adguard.com/dns-query
3. Close and relaunch chrome.

**Expected**
Extension UI can be opened, added sites work through vpn.

**Actual**
Extension UI is empty black, even after 5 mins.
Previously added sutes use direct connection.

**More details**
Extensions works with secure DNS disabled or with other dns provider (cloudflare).
If secure DNS is disabled in chrome but Adguard DNS (DoT/DoH) is configured on router the extension also does not work.

**Comment by maximtop** [comment]
Wasn't able to reproduce it.
@Birbber could you check, please, too?

**Comment by Birbber** [comment]
@deNoor We weren't able to reproduce the issue. Could you please send us a screenshot of console on 'background.html' page? Are there any errors?

**Comment by deNoor** [comment]
:( I saved screenshots on ram disk and restarted before uploading them.
Console window reported fails to connect to api.veryawesomeprivacy.org with "connection_reset" error. And I actually was unable to reach veryawesomeprivacy.org from browser (regular get attempt using a new page) with "connection_reset" error.
Switching to other DNS provider or disabling secure dns allowed to me connect to veryawesomeprivacy.org. After initial load with other dns I was able to set Adguard dns and continue using until next browser restart.

Trying to check more options I exited AdguardVPNForWindows (v1.2.2, socks 5 mode) and was unable to reproduce the issue with extension.
I restarted PC to verify that the issue is gone with windows vpn client unloaded and forgot to save screenshots. But the issue is actually gone, it was not the extension or adguard DNS fault. :)

I already had major network issues with AdguardVPN for windows v1.2.0 (all connections reset attempt in short intervals) and reported them in the related issue tracker.
